braindecode/samplers/base.py

```diff
@@ -123,3 +133,84 @@ def __len__(self):
     def __iter__(self):
         for start_ind in self.start_inds:
             yield tuple(range(start_ind, start_ind + self.n_windows))
+
+
+class RandomSampler(RecordingSampler):
+    """Sample sequences of consecutive windows.
+
+    Parameters
+    ----------
+    metadata : pd.DataFrame
+        See RecordingSampler.
+    n_windows : int
+        Number of consecutive windows in a sequence.
+    seq_nbr : int
+        Number of sequences in the sampler.
+    random_state : np.random.RandomState | int | None
+        Random state.
+    """
+    def __init__(self, metadata, n_windows, seq_nbr=10,
+                 random_state=None):
+        super().__init__(metadata, random_state=random_state)
+
+        self.n_windows = n_windows
+        self.n_windows_stride = 1
+        self.seq_nbr = seq_nbr
+
+        keys = [k for k in ['subject', 'session', 'run', 'target']
+                if k in self.metadata.columns]
+        if not keys:
+            raise ValueError(
+                'metadata must contain at least one of the following columns: '
+                'subject, session or run.')
+        self.info_class = self.metadata.reset_index().groupby(keys)[
+            ['index', 'subject']].agg(['unique'])
+        self.info_class.columns = self.info.columns.get_level_values(0)
+
+    def _compute_seq_start_ind(self, rec_ind=None, class_ind=None):
+        """Randomly compute sequence start indice.
+
+        Choose a window associated with a random recording
+        and a random class and place it randomly in a sequence.
+        The function returns the beginning of this sequence.
+
+        Returns
+        -------
+        start_ind : int
+            The indice of the first windows of possible sequences.
```

Comment:
```suggestion
            The index of the first window of a possible sequence.
```
or make it plural is you return an array and not an int